### PR TITLE
introduce new docker setup

### PR DESCRIPTION
**Pull request by jarkenau**
The update includes: 

- separate docker-compose setups for production and the devcontainer
- using individual Dockerfiles for the Django and Vue applications
- force emulation on arm devices for the devcontiner due to `libpq` error, see https://stackoverflow.com/questions/62807717/how-can-i-solve-postgresql-scram-authentifcation-problem

### CODE REVIEW

**Comment by lucaslootalot** [comment]
@jarkenau The error with the port forwarding could not be reproduced.

File: django/dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ENV PYTHONUNBUFFERED 1
+
+# make sure everthing can be build on arm devices
+RUN apt-get -qq update && apt-get -qq install -y --no-install-recommends -y \
+        gcc \
+        python3-pip \
+        python3-dev \
+        python3.8-venv \
+        libpq-dev \
+        gpg \
```
Comment (Anafabula) [inline django/dockerfile:12]: Do we need gpg here?

File: .devcontainer/Dockerfile.dev [hunk 2]
```diff
@@ -27,14 +25,21 @@ RUN pip3 install pre-commit
 ####################################################
 # Install django
 ####################################################
-RUN pip3 install django==4.0 djangorestframework django-cors-headers
+RUN pip3 install \
+        django==4.0.1\
+        djangorestframework==3.13.1 \
+        django-cors-headers==3.12.0 \
+        django-oauth-toolkit==2.0.0\
+        psycopg2-binary==2.9.3
```
Comment (Anafabula) [inline .devcontainer/Dockerfile.dev:33]: tzdata is listed in requirements.txt but not installed and the django admin console crashes without it 
```suggestion
        psycopg2-binary==2.9.3 \
        tzdata==2022.1
```

**Review by Anafabula — COMMENTED** [review]
Works fine on Linux (except for the tzdata thing)

**Pull request merged**